api/src/main/java/marquez/MarquezApp.java

```diff
@@ -57,7 +60,9 @@ public final class MarquezApp extends Application<MarquezConfig> {
 
   // Monitoring
   private static final String PROMETHEUS = "prometheus";
+  private static final String PROMETHEUS_V2 = "prometheus_v2";
   private static final String PROMETHEUS_ENDPOINT = "/metrics";
+  private static final String PROMETHEUS_ENDPOINT_V2 = "/metrics/v2";
```

Comment:
The standard is usually to expose `/metrics` for metric scraping. For example, in datadog, you set the.`openmetrics_endpoint` to point to `/metrics` to [begin scraping metrics via HTTP](https://docs.datadoghq.com/containers/kubernetes/prometheus/?tab=kubernetesadv2#simple-metric-collection).

I think exposing a separate endpoint provides the benefit of non-breaking changes, but does introduce infra updates which might be a heave lift for some orgs.

Reply:
`v2beta/metrics` used now.